pkgs/tools/misc/tmux-sessionizer/default.nix

```
@@ -5,20 +5,32 @@
 , openssl
 , pkg-config
 , Security
-}:
+, testers
+, tmux-sessionizer
+}: let
 
-rustPlatform.buildRustPackage rec {
-  pname = "tmux-sessionizer";
-  version = "0.2.3";
+  name = "tmux-sessionizer";
+  version = "0.3.0";
```

Don't specify the version number multiple times
```suggestion
  inherit version;
```